charts/newrelic-infra-operator/Chart.yaml

```diff
@@ -7,7 +7,7 @@ sources:
   - https://github.com/newrelic/newrelic-infra-operator
   - https://github.com/newrelic/newrelic-infra-operator/tree/main/charts/newrelic-infra-operator
 
-version: 1.0.11
+version: 2.0.0
 appVersion: 0.7.0
```

Comment:
nit: it would be nice to merge all Prs and release a new version if we have time

Reply:
prs regarding dependencies

Reply:
I'll try to merge all the dependabot PRs and release first.